### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ayvacs**

### CODE REVIEW

**Comment by MrCheeze446** [comment]
A few suggestions: 

- Combine AA and AF under one entry, the two features are similar enough to be listed under one.
- remove fast texture pack switching (old, no longer exists)
- remove time control (old, no longer exists)
- remove autosave (you get the idea)
- remove fullscreen resolution, this has been a Minecraft feature for a long time now
- remove mention of HD textures and fonts, this was added by Minecraft
- remove mention of Vsync, this is vanilla now
- clear water was added in 1.13, probably specify this
- specify that OptiFine just adds buttons for toggle the debugging features such as lagometer, debug profiler, and advanced tooltips
- MCForum is essentially dead at this point, there isn't really any harm in keeping it but I'd personally remove it
- combine Better Grass, Better Snow, Random Mobs, Connected Textures, and Natural textures under one bullet, have each feature listed be its own sub-bullet.

I could be wrong with some of these features I suggest to remove mention of, I'd let sp look over this and confirm what is or is not correct about this.

**Comment by vinc3nz** [comment]
Looks nice!
Suggestion:
- Add a message, that this repo is not Optfine Sourceode.
- Add the Website (optifine.net) and the Backup Website (optifined.net)

**Comment by RaptaG** [comment]
Line 86, `)` is placed at a wrong place, making it part of the link..

**Comment by Ari-RERA-43** [comment]
Since Fast fog has been removed, it might be a good idea to remove that to prevent confusion.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+.DS_Store
```
Comment (ooliver1) [inline .gitignore:2]: Unrelated, place this in your global gitignore file

File: README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<div align="center">
+  <h1>OptiFine</h1>
+
+  OptiFine is a mod which adds support for HD textures, shaders, and a ton of options for better looks and performance. Doubling the FPS is common.
+
+  [r/OptiFine](https://www.minecraftforum.net/linkout?remoteUrl=https%253a%252f%252fwww.reddit.com%252fr%252fOptifine%252f") || [Twitter](https://twitter.com/sp614x) || [Issue Tracker](https://github.com/sp614x/optifine/issues) || [Minecraft Forum](https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1272953-optifine-hd-fps-boost-dynamic-lights-shaders-and)
+
+  <br>
+
+  **[Donate to OptiFine](https://optifine.net/donate)** and receive the [OptiFine cape](http://i.imgur.com/zXIkv.png) as a sign of your awesomeness. Capes are visible to everyone using OptiFine. Thank you for being awesome.
```
Comment (ooliver1) [inline README.md:10]: Bolded text should just go in []

File: README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+<div align="center">
+  <h1>OptiFine</h1>
+
+  OptiFine is a mod which adds support for HD textures, shaders, and a ton of options for better looks and performance. Doubling the FPS is common.
+
+  [r/OptiFine](https://www.minecraftforum.net/linkout?remoteUrl=https%253a%252f%252fwww.reddit.com%252fr%252fOptifine%252f") || [Twitter](https://twitter.com/sp614x) || [Issue Tracker](https://github.com/sp614x/optifine/issues) || [Minecraft Forum](https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1272953-optifine-hd-fps-boost-dynamic-lights-shaders-and)
+
+  <br>
```
Comment (ooliver1) [inline README.md:8]: Why not use other non-html methods for line breaks

**Review by ooliver1 — CHANGES_REQUESTED** [review]

**Comment by TheKodeToad** [comment]
Some text mentioning that this repo is primarily for issue tracking and docs would also be nice.

**Pull request closed without merging**